wordpress-mobile/WordPress-iOS
# Remove FeatureFlag.modernizeMedia and associated code

To test: 

- Verify that Site Media screens opens
- Verify that Site Media pickers in Gutenberg, Site Icon flow, and Feature Image flow allow you to select images from Site Media

I wish I could remove more, but Aztec and Story Editor use WPMediaPicker and I'm hoping we get to the point where both can be removed.

## Regression Notes
1. Potential unintended areas of impact: n/a (only removal)
2. What I did to test those areas of impact (or what existing automated tests I relied on): n/a
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
